arkane/statmech.py

```diff
@@ -1395,3 +1396,125 @@ def readGjf(self):
 
 
 
+   def writeXYZ(self):
+       """
+       write an .xyz file for Q2DTor
+       done based on the angle coordinates (0.0,0.0)
+       """
+       atdict = {x.number:x.symbol for x in elementList}
+       for i in xrange(len(self.phi1s)):
```

Comment:
That one doesn't make sense to me...why call a function that produces tuples with output you don't need instead of calling one that produces exactly the tuples you need?